sippy-ng/src/component_readiness/TriagedRegressions.js

```diff
@@ -72,6 +74,29 @@ export default function TriagedRegressions({
     }
   }
 
+  // Helper function to check if triage has any regressions with status -1000
+  const hasStatus1000Regression = (triage) => {
```

Comment:
It would be nice if we could have one common function that both places utilized. Can we combine these and put it in CompReadyUtils?